src/yunohost/app.py

```diff
@@ -993,16 +994,41 @@ def app_remove(operation_logger, app):
     os.system('chown -R admin: /tmp/yunohost_remove')
     os.system('chmod -R u+rX /tmp/yunohost_remove')
 
-    args_list = [app]
+    with open(os.path.join(APPS_SETTING_PATH, app, 'manifest.json')) as json_manifest:
+        manifest = json.load(json_manifest)
```

Comment:
To be kept in mind that this should be replaced by the `get_manifest_for_app` thing from the other toml manifest PR